pkg/volume/util/operationexecutor/operation_generator.go

```diff
@@ -73,14 +79,24 @@ type operationGenerator struct {
 }
 
 // NewOperationGenerator is returns instance of operationGenerator
-func NewOperationGenerator(kubeClient clientset.Interface,
+func NewOperationGenerator(
+	kubeResourcePartitionClient clientset.Interface,
 	volumePluginMgr *volume.VolumePluginMgr,
 	recorder record.EventRecorder,
 	checkNodeCapabilitiesBeforeMount bool,
-	blkUtil volumepathhandler.BlockVolumePathHandler) OperationGenerator {
+	blkUtil volumepathhandler.BlockVolumePathHandler,
+	kubeTenantPartitionClients ...[]clientset.Interface) OperationGenerator {
```

Comment:
this code is also used in controller. hence the "optional" variable kubeTenantPartitionClients which is only provided by the kubelet, not the controller

Reply:
also just a comment here. resolving.